pootle/apps/pootle_app/management/commands/config.py

```diff
@@ -0,0 +1,236 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) Pootle contributors.
+#
+# This file is a part of the Pootle project. It is distributed under the GPL3
+# or later license. See the LICENSE file for a copy of the license and the
+# AUTHORS file for copyright and authorship information.
+
+import collections
+import json
+import os
+
+os.environ['DJANGO_SETTINGS_MODULE'] = 'pootle.settings'
+
+from django.contrib.contenttypes.models import ContentType
+from django.core.exceptions import FieldError
+from django.core.management.base import BaseCommand, CommandError
+from django.utils.translation import ugettext_lazy as _
+
+from pootle.core.delegate import config
+
+
+class Command(BaseCommand):
+    help = "Manage configuration for Pootle objects."
+
+    def add_arguments(self, parser):
+        super(Command, self).add_arguments(parser)
+        parser.add_argument(
+            'content_type',
+            action='store',
+            nargs="?",
+            help=_("Get/set config for given content type"))
+        parser.add_argument(
+            'object',
+            action='store',
+            nargs="?",
+            help=_("Unique identifier for object, by default pk"))
+        parser.add_argument(
+            '-o', '--object-field',
+            action='store',
+            dest='object_field',
+            help=_(
+                "Use given field as identifier to get object, "
+                "should be unique"))
+        parser.add_argument(
+            '-g', '--get',
+            action='store',
+            dest='get',
+            metavar="KEY",
+            help=_("Get config with this key, key must be unique"))
+        parser.add_argument(
+            '-l', '--list',
+            action='append',
+            dest='list',
+            metavar="KEY",
+            help=_(
+                "List config with this key, "
+                "can be specified multiple times"))
+        parser.add_argument(
+            '-s', '--set',
```

Comment:
`--get` and `--set` look like mutually exclusive options, so perhaps it is better to put them in a [argparse mutually exclusive group](https://docs.python.org/2/library/argparse.html#mutual-exclusion).